static/templates/invitation_failed_error.hbs

```diff
@@ -1,4 +1,11 @@
-<p id="invitation_error_message">{{ error_message }}</p>
+<p id="invitation_error_message">{{ error_message }}
+    {{#if max_limit_reached}}
+        {{#tr}}
+            You can also <z-link>add users with a reusable invite link</z-link>.
+            {{#*inline "z-link"}}<a href="https://zulip.com/help/invite-new-users#send-invitations">{{> @partial-block}}</a>{{/inline}}
```

Comment:
@alya thoughts on the precise string here?  The approachj lgtm.

Reply:
Seems good! Maybe add a blank line before the "Please contact..." part?

Reply:
I tried adding blank line, but not able to do it properly.
I added `\n\n` in the message sent from backend but due to HTML considering a single space only instead of blank line, it does not change anything. To override this behavior, I tried setting `white-space` property to `pre-wrap` and `pre-line` but this also moves the `You can also ...` text to next line as in the below screenshot.
![Screenshot from 2021-09-22 16-03-57](https://user-images.githubusercontent.com/35494118/134336793-1e7489f1-e670-4f50-aede-ec518094c8e7.png)
One way to do this is just move the `Please contact ..` part to be only in frontend and not in the error message returned by API. @timabbott suggestions?

Reply:
Yeah, I think having frontend-only strings for the extended error messages is totally reasonable.  We don't really support links/formatting in the basic error messages generated by the server.

Reply:
(I think it'd be fine for the API error to just say you don't have enough for today)

Reply:
How do we want to pass the zulip administrator email to client in such case. Do we want to send in the `register` response or we should send it as a data field in error. We do not use this field anywhere else and we do not send such fields in data fields of error in any other case.